org.uqbar.project.wollok.game.ui/src/org/uqbar/project/wollok/game/ui/commons/ZipHandler.java

```diff
@@ -0,0 +1,187 @@
+package org.uqbar.project.wollok.game.ui.commons;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipException;
+import java.util.zip.ZipFile;
+import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
+
+import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IAdaptable;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.IPackageFragmentRoot;
+import org.eclipse.jdt.core.JavaCore;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.PlatformUI;
+
+public class ZipHandler {
+	List<File> fileList;
+
+	public ZipHandler() {
+		this.fileList = new ArrayList<File>();
+	}
+
+	public void addFileToZip(File aFile) {
+		this.fileList.add(aFile);
+	}
+
+	public String getProjectPath() {
+		return ResourcesPlugin.getWorkspace().getRoot()
+				.findMember(this.getProjectName()).getLocation().toString();
+	}
+
+	public IPath getProjectIPath() {
+		return ResourcesPlugin.getWorkspace().getRoot()
+				.findMember(this.getProjectName()).getLocation();
+	}
+
+	public void makeZipFile(String fileOutput) throws FileNotFoundException,
+			IOException {
+		FileOutputStream fos = new FileOutputStream(fileOutput);
+		ZipOutputStream zos = new ZipOutputStream(fos);
+		for (int i = 0; i < fileList.size(); i++) {
+			File fileToAdd = fileList.get(i);
+			if (fileToAdd.getName().contains(".json"))
+				addToZipFile("", fileToAdd, zos);
+			else if (fileToAdd.getName().contains(".wlk")
+					|| fileToAdd.getName().contains(".wpgm"))
```

Comment:
Theres a class in wollok which holds constants for file extensions. This avoids having the string all over the places. WollokConstants.xtend